srcpkgs/snappy/patches/inline.patch

```diff
@@ -0,0 +1,5 @@
+--- snappy.cc	2021-05-05 02:53:34.000000000 +0400
++++ snappy.cc	2021-05-24 01:24:59.124654893 +0400
+@@ -1017 +1017 @@ SNAPPY_ATTRIBUTE_ALWAYS_INLINE
+-size_t AdvanceToNextTag(const uint8_t** ip_p, size_t* tag) {
++inline size_t AdvanceToNextTag(const uint8_t** ip_p, size_t* tag) {
```

Comment:
Is this upstream? Why is it needed? (Add this info as a comment in the patch before the diff.

Reply:
there's an open PR about it https://github.com/google/snappy/pull/128/commits/0c716d435abe65250100c2caea0e5126ac4e14bd

Reply:
Ok, please link the PR in that commit, since it isn't upstream yet.